services/ui-src/src/containers/PackageList.test.js

```diff
@@ -152,34 +150,34 @@ it("does not display waiver rais", async () => {
   ).toBeDisabled();
 });
 
-it.each`
-  filterFieldType    | filterFieldValue                      | inName                 | inValue          | textShown
-  ${"currentStatus"} | ${Workflow.ONEMAC_STATUS.WITHDRAWN}   | ${"clockEndTimestamp"} | ${null}          | ${"N/A"}
-  ${"currentStatus"} | ${Workflow.ONEMAC_STATUS.TERMINATED}  | ${"clockEndTimestamp"} | ${null}          | ${"N/A"}
-  ${"currentStatus"} | ${Workflow.ONEMAC_STATUS.UNSUBMITTED} | ${"clockEndTimestamp"} | ${null}          | ${"Pending"}
-  ${"currentStatus"} | ${Workflow.ONEMAC_STATUS.IN_REVIEW}   | ${"clockEndTimestamp"} | ${null}          | ${"Pending"}
-  ${"currentStatus"} | ${Workflow.ONEMAC_STATUS.SUBMITTED}   | ${"clockEndTimestamp"} | ${1570378876000} | ${"Pending"}
-  ${"currentStatus"} | ${Workflow.ONEMAC_STATUS.RAI_ISSUED}  | ${"clockEndTimestamp"} | ${null}          | ${"Clock Stopped"}
-`(
-  "shows $textShown in $inName when $filterFieldType is $filterFieldValue and value is $inValue",
-  async ({ filterFieldType, filterFieldValue, inName, inValue, textShown }) => {
-    const testPackage = packageList[0];
-    testPackage[filterFieldType] = filterFieldValue;
-    testPackage[inName] = inValue;
-    PackageApi.getMyPackages.mockResolvedValue([testPackage]);
-
-    render(<PackageList />, { wrapper: ContextWrapper });
-
-    // wait for loading screen to disappear so package table displays
-    await waitForElementToBeRemoved(() => screen.getByTestId(LOADER_TEST_ID));
-
-    // get the row for the status
-    const packageRow = within(
-      screen.getAllByText(filterFieldValue)[0].closest("tr")
-    );
-    expect(packageRow.getAllByText(textShown)[0]).toBeInTheDocument();
-  }
-);
+// it.each`
```

Comment:
Part of me wants to delete this, but.... it is actually a pretty good example of doing this kind of compound test with different data and I'm pretty sure I've copied it before :)

Reply:
yeah i really liked it as an example too and was going to make a comment about why i left it in...but forgot. :)